Classes/gitx_askpasswd_main.m

```diff
@@ -171,7 +171,10 @@ int	main( int argc, const char** argv )
 		
 		[defaults synchronize];
 		
-		return code;
+		if (code <= INT_MAX) {
+			return (int)code;
+		};
+		return INT_MAX;
```

Comment:
Oh, so it can return other values. I guess I didn't finish reading the documentation page I linked to. 😣 The new check is perfect!